.github/workflows/test.yml

```diff
@@ -0,0 +1,42 @@
+name: Tests
+on:
+  pull_request:
+    branches:
+      - "**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
```

Comment:
Why do we need a timeout?

Reply:
To prevent long running workflows. 
Saw this being used in the website-backend workflow also.

Reply:
> To prevent long running workflows. Saw this being used in the website-backend workflow also.

what is the parameter to decide 5 minutes